app/Command/Docs.hs

```diff
@@ -72,6 +79,15 @@ docgen (PSCDocsOptions fmt inputGlob) = do
                 >>= uncurry D.convertTaggedModulesInPackage)
     >>= successOrExit
 
+  writeTagsToFile :: String -> [String] -> IO ()
+  writeTagsToFile outputFilename tags = do
+    currentDir <- getCurrentDirectory
+    let outputFile = currentDir </> outputFilename
+    let pattern = compile outputFilename
+    let text = T.pack . unlines $ tags
+    globDir1 pattern currentDir >>= mapM_ removeFile
```

Comment:
I don’t think we need to do any globbing here, won’t the file just get overwritten if it already exists?